mathics/builtin/system.py

```diff
@@ -596,16 +596,20 @@ class Share(Builtin):
 
     def apply_0(self, evaluation) -> Integer:
         """Share[]"""
+        totalmem = psutil.virtual_memory().available
```

Comment:
psutil is currently an _optional_ requirement.

Reply:
OK, so I can put this version in an "if" statement, and update the documentation accordingly.